components/test_raftstore/src/pd.rs

```diff
@@ -562,7 +562,7 @@ impl Cluster {
                         .all(|x| x.get_store_id() != *store_id)
                     {
                         let peer = Either::Left(new_peer(*store_id, self.alloc_id().unwrap()));
-                        let policy = SchedulePolicy::Repeat(1);
+                        let policy = SchedulePolicy::TillSuccess;
```

Comment:
Why?

Reply:
Generally it will always be retried until success.

Reply:
And every retry will allocate a new peer id, which will be rejected by conf change proposal check.

Reply:
What if it can never succeed? For example, an old operator adds a peer on the store and occupy the store id with a different peer id.